README.md

```diff
@@ -145,17 +146,18 @@ There are 2 test classes provided:
+| [`code/dayofyear.m`](code/dayofyear.m)                                 | The [`dayofyear`](code/dayofyear.m) function returns the day-of-year number for a given date string "mm/dd/yyyy"                                                                                     |
+| [`tests/TestExamples.m`](tests/TestExamples.m)                         | The [`TestExamples`](tests/TestExamples.m) class provides a few equality and negative tests for the [`dayofyear`](code/dayofyear.m) function                                                         |
+| [`tests/ParameterizedTestExample.m`](tests/ParameterizedTestExample.m) | The [`ParameterizedTestExample`](tests/ParameterizedTestExample.m) class provides 12 tests for the [`dayofyear`](code/dayofyear.m) function using the parameterized test format                      |
+| [`azure-pipelines.yml`](###Azure-DevOps)                               | The [`azure-pipelines.yml`](azure-pipelines.yml) file defines the pipeline that runs on [Azure DevOps](https://marketplace.visualstudio.com/items?itemName=MathWorks.matlab-azure-devops-extension). |
+| [`.circleci/config.yml`](###CircleCI)                                  | The [`config.yml`](.circleci/config.yml) file defines the pipeline that runs on [CircleCI](https://circleci.com/orbs/registry/orb/mathworks/matlab)                                                  |
+| [`.github/workflows/ci.yml`](###GitHub-Actions)                        | The [`ci.yml`](.github/workflows/ci.yml) file defines the pipeline that runs on [GitHub Actions](https://github.com/matlab-actions/overview)                                                         |
+| [`Jenkinsfile`](###Jenkins)                                            | The [`Jenkinsfile`](Jenkinsfile) file defines the pipeline that runs on [Jenkins](https://plugins.jenkins.io/matlab/)                                                                                |
+| [`.travis.yml`](###Travis-CI)                                          | The [`.travis.yml`](.travis.yml) file defines the pipeline that runs on [Travis CI](https://docs.travis-ci.com/user/languages/matlab/)                                                               |
+| [`.gitlab-ci.yml`](###GitLab-CI/CD)                                    | The [`.gitlab-ci.yml`](.gitlab-ci.yml) file defines the pipeline that runs on [GitLab CI/CD](https://docs.gitlab.com/ee/ci/)                                                                         |
+| [`.teamcity/settings.kts`](###TeamCity)                                | The [`.teamcity/settings.kts`](.teamcity/settings.kts) file defines the pipeline that runs on [TeamCity](https://www.jetbrains.com/help/teamcity/kotlin-dsl.html)                                                             |
```

Comment:
I am somewhat confused, @nbhoski, because haven't documented pipelines for TeamCity: https://github.com/mathworks/matlab-teamcity-plugin/blob/main/CONFIGDOC.md. Isn't it out of model to expose an undocumented functionality in this table? Unlike this new entry, all existing table entries point to documented functionality.

Reply:
I agree, Would we want to publish it in teamcity first ? However this DSL support is implicit we did not do anything extra to support it. It would be more of documenting on how to use this as DSL code as mentioned here https://www.jetbrains.com/help/teamcity/kotlin-dsl.html